src/server.ts

```diff
@@ -173,10 +174,31 @@ apolloServer.applyMiddleware({
 });
 apolloServer.installSubscriptionHandlers(httpServer);
 
+const logWarningForSuperAdminEnvVariable = async () => {
+  const superAdminExist = await User.exists({ userType: "SUPERADMIN" });
+  const isVariablePresentInEnvFile = !!LAST_RESORT_SUPERADMIN_EMAIL;
+  if (superAdminExist) {
+    if (isVariablePresentInEnvFile) {
+      logger.warn(
+        "\x1b[1m\x1b[33m%s\x1b[0m",
+        "You have created super admin, please remove the LAST_RESORT_SUPERADMIN_EMAIL variable from .env file if you don't require it"
```

Comment:
The LAST_RESORT_SUPERADMIN_EMAIL variable configured in your .env file poses a security risk. We strongly recommend that you remove it if not required. Please refer to the documentation in the INSTALLATION.md file.

Reply:
done